### ISSUE TITLE
[Open Path] ...

### ISSUE THREAD

**Issue by nukesean** [raycast/extensions#7020]
### Extension

https://www.raycast.com/koinzhang/open-path

### Description

The extension works perfectly for everything except URL schemes. Specifically, whenever the command is invoked to open a deeplink, it just searches Google for the URL, instead. See demo here: [https://imgur.com/a/QBfExVk](https://imgur.com/a/QBfExVk).

### Steps To Reproduce

1. Select deeplink
2. Execute extension

**Comment by koinzhang** [comment]
Hi @nukesean. Thanks for your feedback, I mistakenly removed the code to open deeplink in a previous update and have now submitted a PR #7022 to fix this feature.
If you have any other questions, feel free to give feedback

**Comment by nukesean** [comment]
Awesome! Thank you, @koinzhang! I do have one more issue to report, though.

After I reported the deeplink bug, I discovered that the extension does not handle IP addresses correctly if there is a path afterwards. For example `140.82.113.4` works, but `140.82.113.4/raycast/extensions/issues/7020` does not; it just opens up a tab at `about:blank` for some reason. It _does_ work, however, if the full path also includes the `https://` at the beginning.
